src/providers/BackgroundReplacementProvider/index.tsx

```diff
@@ -179,10 +182,28 @@ export const BackgroundReplacementProvider: FC<
     }
   };
 
+  const changeBackgroundReplacementImage = async (
+    imageBlob: Blob
+  ) => {
+    if (!backgroundReplacementProcessor) {
+      logger.warn(`BackgroundReplacementProcessor has not been initialized yet.`);
+      return;
+    }
+    try {
+      await backgroundReplacementProcessor.setImageBlob(imageBlob);
+      logger.info(`Background replacement image changed to new imageBlob: ${imageBlob}`);
+    } catch (error) {
+      throw new Error(
```

Comment:
I’m just asking for you opinion if we should notify consumer, nor enforcing you. 😂 You may want to use Promise if throwing error.

Reply:
What is the criteria for throwing error/logging error? As for this use case, if there is error, then we keep use the previous image. Then I think we do not need to throw error.

Reply:
2 factors I can think of:
- check how other api handles error in this repo
- Is there any related customer ask for the failure notification, customer may want to notify the end-users if failed to apply the image.

Reply:
Don't think you are using Promise correctly...

Reply:
I have tested the code with my changes in demo app, what do you mean by not correct?